docs/features/techdocs/getting-started.md

```diff
@@ -270,8 +270,11 @@ You can do so by including the following lines right above `USER node` of your
 `Dockerfile`:
 
 ```Dockerfile
-RUN apt-get update && apt-get install -y python3 python3-pip
+RUN apt-get update && apt-get install -y python3 python3-pip gcc musl-dev openjdk-17-jdk curl graphviz fonts-dejavu fontconfig
```

Comment:
Might be nice to make the plantuml version a variable with some instructions on how to bump it. These are the partial parts of the custom techdocs Dockerfile we use. On that note and improvement to the instructions would be to have one line script that can get the SHA value.

```Dockerfile
# To update plantuml version:
#   1. Check the latest version from https://plantuml.com/download
#   2. Download the updated version from http://sourceforge.net/projects/plantuml/files/plantuml.[PLANTUML_VER].jar/download
#   3. Run `sha1sum plantuml.[PLANTUML_VER].jar` to get the checksum on the file you downloaded
#   4. Update the version & checksum args
ARG PLANTUML_VER="1.xxxx.x"
ARG PLANTUML_CHECKSUM="xxxxxxxx"

# ...

# Download plantuml file, Validate checksum & move plantuml file
RUN curl -o plantuml.jar -L "http://sourceforge.net/projects/plantuml/files/plantuml.$PLANTUML_VER.jar/download" \
  && echo "$PLANTUML_CHECKSUM  plantuml.jar" | sha1sum -c - \
  && mv plantuml.jar /opt/plantuml.jar
```

Reply:
So, if we wanted to switch to using GH releases, the following bash will get the latest version/checksum:

```sh
VERSION="$(curl --silent "https://api.github.com/plantuml/plantuml/releases/latest" | grep '"tag_name":' | sed -E 's/.*"([^"]+)".*/\1/' | sed -E 's/v([0-9.]+).*/\1/')"
CHECKSUM="$(curl -sSL "https://github.com/plantuml/plantuml/releases/download/v$VERSION/plantuml-$VERSION.jar" | sha1sum | cut -d ' ' -f 1)"
echo "PlantUML (Latest): $VERSION (checksum $CHECKSUM)"
```

Reply:
@andrewthauer thanks for the input! I have updated the PR.